feature-libs/asm/components/customer-list/customer-list.component.ts

```diff
@@ -39,6 +39,7 @@ export class CustomerListComponent implements OnInit, OnDestroy {
     trap: true,
     block: true,
     autofocus: 'customer-list-selector',
+    ignoreArrows: true,
```

Comment:
Can this be changed to `ignoreArrowKeys`?
Or, more precisely, `ignoreUpDownArrowKeys`?

Reply:
yea, ok